lib/ae_mdw/sync/async_tasks/update_aex9_presence.ex

```diff
@@ -24,10 +25,19 @@ defmodule AeMdw.Sync.AsyncTasks.UpdateAex9Presence do
 
     Log.info("[update_aex9_presence] #{inspect(contract_pk)} after #{time_delta / @microsecs}s")
 
-    :mnesia.sync_transaction(fn ->
-      Enum.each(amounts, fn {{:address, account_pk}, _amount} ->
-        Contract.aex9_write_new_presence(contract_pk, -1, account_pk)
+    contract_id = :aeser_api_encoder.encode(:contract_pubkey, contract_pk)
+    create_txi = Origin.tx_index({:contract, contract_id})
+
+    if create_txi do
+      :mnesia.sync_dirty(fn ->
+        Enum.each(amounts, fn {{:address, account_pk}, _amount} ->
+          Contract.aex9_write_new_presence(contract_pk, create_txi, account_pk)
+        end)
       end)
-    end)
+    else
+      Log.error("Missing create_txi for contract #{contract_id}")
```

Comment:
This isn't really an error. There's many contracts that don't have a create txi

Reply:
For now aex9 contracts is only indexed after a `:contract_create_tx` (with a `create_txi`).
Good observation though that Chain.create and Chain.clone should index aex9 contracts too. 
Issue opened #425

Reply:
For now aex9 contracts are only indexed after a `:contract_create_tx` (with a `create_txi`).
Good observation though that Chain.create and Chain.clone should index aex9 contracts too. 
Issue opened #425